test/integration/targets/collections/collections/ansible_collections/testns/content_adj/plugins/cache/custom_jsonfile.py

```diff
@@ -52,7 +52,6 @@ class CacheModule(BaseFileCacheModule):
     """
     A caching module backed by json files.
     """
-    NAME = 'testns.content_adj.custom_jsonfile'
```

Comment:
If we require cache plugins to set a load name, then they also aren't portable between collections as NAME would need to be tailored to the collection name. The more I look at this the more I like using _load_name. I think inventory plugins should also get rid of NAME for the same reason.